status.md

```diff
@@ -45,17 +45,17 @@ developments occur.
-| auth.<span class="grayed">getpebble.com</span><br/>auth-api.<span class="grayed">getpebble.com</span> | Authorization endpoints. | May not be necessary given a new mobile app. |
-| dev-portal.<span class="grayed">getpebble.com</span> | Developer portal. | Replacement will be a part of The Rebble Store. |
+| boot.<span class="grayed">getpebble.com</span> | Provides API endpoints to clients. | Functionality supplanted by Rebble Web Services. |
+| auth.<span class="grayed">getpebble.com</span><br/>auth-api.<span class="grayed">getpebble.com</span> | Authorization endpoints. | Functionality supplanted by Rebble Web Services. |
+| dev-portal.<span class="grayed">getpebble.com</span> | Developer portal. | Mirror available [here](https://developer.rebble.io/developer.pebble.com/index.html). |
 | pb-collector.<span class="grayed">getpebble.com</span> | ??? | |
-| lp.<span class="grayed">getpebble.com</span> | Language packs. | Low priority. Format seems to be mostly understood. |
-| apps.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | App Store. | To be replaced by [The Rebble Store](https://github.com/pebble-dev/rebble-store/). |
-| help.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | Support pages. | Not in immediate danger. Archive.org should suffice. |
-| cloudpebble.net | Cloud-based IDE. | CloudPebble is open source and will continue as a community project once it once it goes down. **_However_**, you should definitely [export your CloudPebble projects for local safekeeping!](https://www.reddit.com/r/pebble/comments/5h61om/bulk_cloudpebble_project_export/) |
-| timeline-sync.<span class="grayed">getpebble.com</span><br/>timeline-api.<span class="grayed">getpebble.com</span> | Timeline servers. | Will continue to be maintained for an indefinite amount of time. Community replacement possible, but very low-priority. |
+| lp.<span class="grayed">getpebble.com</span> | Language packs. | Functionality supplanted by Rebble Web Services. |
+| apps.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | App Store. | Functionality supplanted by Rebble Web Services (apps.rebble.io). |
+| help.<span class="grayed">getpebble.com</span><br/>appstore-api.<span class="grayed">getpebble.com</span> | Support pages. | Mirror available [here](http://pebble-help-legacy.rebble.io/help.getpebble.com/indexd41d.html). |
+| cloudpebble.net | Cloud-based IDE. | CloudPebble is open source and will continue as a community project once it once it goes down. **_However_**, you should definitely [export your CloudPebble projects for local safekeeping!](https://www.reddit.com/r/pebble/comments/5h61om/bulk_cloudpebble_project_export/) Unclear if it'll ever come back as a part of Rebble Web Services. |
```

Comment:
I'm not crazy about this language - can we come up with something less casual/more "scientific" ? ⚗️

Reply:
To be completely honest I have no idea about the specifics - I didn't want to go out on a limb and say anything that wasn't correct - so some pointers here from someone who knows more about the status and integration of all those moving parts would be very helpful!